src/components/BulkUploadListItem.tsx

```diff
@@ -6,29 +6,24 @@ import {
 	ClockIcon,
 	XCircleIcon,
 } from '@heroicons/react/24/outline';
+import { BulkUpload } from '@pdc/sdk';
 import { localizeDateTime, relativizeDateTime } from '../utils/datetime';
-import type { ApiBulkUpload } from '../pdc-api';
 import { EmailLink } from './EmailLink';
 import './BulkUploadListItem.css';
 
-enum BulkUploadStatuses {
-	PENDING = 'pending',
-	IN_PROGRESS = 'in_progress',
-	COMPLETED = 'completed',
-	CANCELED = 'canceled',
-	FAILED = 'failed',
-}
-
 interface BulkUploadListItemProps {
-	upload: ApiBulkUpload;
+	upload: BulkUpload;
 }
 
 export const BulkUploadListItem = ({ upload }: BulkUploadListItemProps) => {
 	const { id, fileName, fileSize, status, createdAt } = upload;
 
 	const relativeTimestamp = (
-		<time dateTime={createdAt} title={localizeDateTime(createdAt)}>
-			{relativizeDateTime(createdAt)}
+		<time
+			dateTime={createdAt?.toString()}
+			title={localizeDateTime(createdAt ?? '')}
```

Comment:
Innnnteresting -- I'm actually surprised that `createdAt` is nullable; it ought not be!

Reply:
Looks like several fields are incorrectly marked as optional in our swagger spec -- I'm opening an issue for this.

Reply:
Issue opened: https://github.com/PhilanthropyDataCommons/service/issues/987

I would actually wait for this to get merged / the new SDK to be released, since it will mean you don't have to make these checks.

Reply:
This is fantastic news! Thank you for checking, yeah I had felt that it was odd that so many of these fields which, if they were not defined, we would want to ignore the object entirely, would be left as optional. Excited to refactor this PR with these changes

Reply:
@hminsky2002 `main` now has the updated SDK with the improved types. 🎉 Feel free to rebase and carry on.